### ISSUE TITLE
Auto Sorting by modification date doesn't work. Files are sorted by accessed date instead.

### ISSUE THREAD

**Issue by oscarus** [linuxmint/nemo#889]
On Nemo's preferences, under the Default View section, if one chooses to arrange items by modification date, the files will sort by accessed date instead.

For a while I noticed something wasn't right when the files on a new folder wouldn't be immediately sorted and they would move around in a seemingly random order until the thumbnails were cached. I also noticed that none of the four default columns had the little arrow that indicated how the files were sorted. I then added a "date accessed" column, and that's when I noticed the little sorting arrow under that column.

Basically, sorting by name and size by default works as expected. Selecting "by modification date" sorts files by date accessed and selecting "by access date"  appears to sort nothing. I didn't check if the other sorting preferences work properly, but it might be worth looking into it just in case.

**Cinnamon 17.1**
**nemo 2.4.5**
